.vsts-dotnet.yml

```
@@ -107,7 +107,7 @@ extends:
         - name: VisualStudio.MajorVersion
           value: 17
         - name: VisualStudio.ChannelName
-          value: 'int.main'
+          value: 'int.d17.8'
```

Add a merge conflict here, maybe something like

```suggestion
          value: 'int.d17.8' # MSBuild's vs17.8 branch corresponds to this VS channel
```

So that (once this is in all release branches) flow from release branches to main doesn't bring the last release channel forward (and main stays at `main`).